<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>salaDefecto</class>
 <widget class="QMainWindow" name="salaDefecto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEnviar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="campoTexto">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="chat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>441</width>
      <height>271</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenido a la sala:</string>
    </property>
   </widget>
   <widget class="QLabel" name="salaActual">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuinicio">
    <property name="title">
     <string>inicio</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionCrear_Sala"/>
    <addaction name="actionCambiar_de_Sala"/>
    <addaction name="actionSalir_de_la_Sala"/>
    <addaction name="actionDesconectar"/>
    <addaction name="actionListar_Salas"/>
    <addaction name="actionUsuarios_conectados"/>
   </widget>
   <addaction name="menuinicio"/>
   <addaction name="menuOpciones"/>
  </widget>
  <action name="actionCrear_Sala">
   <property name="text">
    <string>Crear Sala</string>
   </property>
  </action>
  <action name="actionDesconectar">
   <property name="text">
    <string>Desconectar</string>
   </property>
  </action>
  <action name="actionCambiar_de_Sala">
   <property name="text">
    <string>Cambiar de Sala</string>
   </property>
  </action>
  <action name="actionSalir_de_la_Sala">
   <property name="text">
    <string>Salir de la Sala actual</string>
   </property>
   <property name="iconText">
    <string>Salir de la sala actual</string>
   </property>
   <property name="toolTip">
    <string>Salir de la sala actual</string>
   </property>
  </action>
  <action name="actionListar_Salas">
   <property name="text">
    <string>Listar Salas</string>
   </property>
  </action>
  <action name="actionUsuarios_conectados">
   <property name="text">
    <string>Usuarios conectados</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
